pkg/api/types.go

```diff
@@ -239,6 +241,18 @@ type SecretVolumeSource struct {
 	Target ObjectReference `json:"target"`
 }
 
+// NFS represents and NFS Mount that lasts the lifetime of a pod
+type NFSVolumeSource struct {
+	// Server is the hostname or ip address of the NFS server
+	Server string `json:"server"`
+
+	// SourcePath is the exported NFS share
+	SourcePath string `json:"sourcePath"`
+
+	// MountOptions is a comma delimited list of mount options
+	MountOptions string `json:"mountOptions"`
```

Comment:
Sure thing. The mount options is just the string of options passed to the mount command via the -o flag. I'll add more documentation inline.

Options are specified with a -o flag followed by a comma separated string of options.